CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `custom/text`: The `content` setting and all its properties are deprecated in favor of `format` with the same functionality. ([`#2676`](https://github.com/polybar/polybar/pull/2676))
 
 ### Added
+-  Support for actions on hover via `mousebtn::HOVER_START` and `mousebtn::HOVER_END` ([`#1064`](https://github.com/polybar/polybar/issues/1064), [`#2868`](https://github.com/polybar/polybar/pull/2868)).
```

Comment:
The changelog is more for users than developers and `mousebtn::HOVER_START` doesn't really mean anything to them. Better to list the numbers corresponding to the two new actions.